library/Zend/Filter/StringToLower.php

```diff
@@ -53,7 +53,7 @@ public function filter($value)
         }
         $value = (string) $value;
 
-        if ($this->options['encoding'] !== null) {
+        if (null !== $this->getEncoding()) {
```

Comment:
The yoda condition is cosmetic indeed, but so is PSR2 :wink: Personally I prefer yoda conditions, because I think defensive programming is good practice.